File: DDS_Restaurant_Solution/DDS_Restaurant_Solution/Models/Empleado.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DDS_Restaurant_Solution.Models
{
    public class Empleado
    {
        [MaxLength(13),Key]
        public string numIdentidad { get; set; }

        [MaxLength(25)]
        public string nombre { get; set; }

        [MaxLength(35)]
        public string apellidos { get; set; }

        [MaxLength(50)]
        public string direccion { get; set; }

        [Column(TypeName = "Date")]
        public DateTime fechaNacimiento { get; set; }

        [MaxLength(10)]
        public string telefono { get; set; }

        [MaxLength(10)]
        public string celular { get; set; }
        
        [MaxLength(25)]
        public string correElectronico { get; set; }

        public int idMunicipio { get; set; }
        [ForeignKey("idMunicipio")]
        public Municipio Municipio { get; set; }

        public int idGenero { get; set; }
        [ForeignKey("idGenero")]
        public Genero Genero { get; set; }

        public int idCargo { get; set; }
        [ForeignKey("idCargo")]
        public Cargo Cargo { get; set; }

        public int idSucursal { get; set; }
        [ForeignKey("idSucursal")]
        public Sucursal Sucursal { get; set; }
    }
}

File: DDS_Restaurant_Solution/DDS_Restaurant_VB/Models/Empleado.vb
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.ComponentModel.DataAnnotations
Public Class Empleado
    <MaxLength(13), Key>
    Public Property numIdentidad As String
    <MaxLength(25)>
    Public Property nombre As String
    <MaxLength(35)>
    Public Property apellidos As String
    <MaxLength(50)>
    Public Property direccion As String
    <Column(TypeName:="Date")>
    Public Property fechaNacimiento As DateTime
    <MaxLength(10)>
    Public Property telefono As String
    <MaxLength(10)>
    Public Property celular As String
    <MaxLength(25)>
    Public Property correElectronico As String
    Public Property idMunicipio As Integer
    <ForeignKey("idMunicipio")>
    Public Property Municipio As Municipio
    Public Property idGenero As Integer
    <ForeignKey("idGenero")>
    Public Property Genero As Genero
    Public Property idCargo As Integer
    <ForeignKey("idCargo")>
    Public Property Cargo As Cargo
    Public Property idSucursal As Integer
    <ForeignKey("idSucursal")>
    Public Property Sucursal As Sucursal
End Class
